src/passes/identifierMangler.ts

```diff
@@ -75,7 +75,9 @@ export class IdentifierMangler extends ASTMapper {
       node.memberName = declaration.name;
     }
   }
-
+  visitStructDefinition(_node: StructDefinition, _ast: AST): void {
+    return;
```

Comment:
This could probably do with a comment explaining the reasoning behind the explicit lack of recursion here

Reply:
Comment Added